frontend/src/components/common/DoseResponsePlot.js

```diff
@@ -9,14 +9,15 @@ import {getConfig} from "../../constants/plotting";
 @observer
 class DoseResponsePlot extends Component {
     render() {
-        const {layout, data} = this.props;
+        const {layout, data, store} = this.props;
 
         return (
             <Plot
                 layout={toJS(layout)}
                 data={toJS(data)}
                 config={getConfig()}
                 style={{width: "100%"}}
+                onRelayout={e => store.saveUserPlotSettings(e)}
```

Comment:
no need to pass the whole store; just pass the relayout function to call. You also missed the `<DoseResponsePlot/>` on the modals, even though you marked the store as a required prop.